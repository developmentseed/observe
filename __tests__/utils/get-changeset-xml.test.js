/* global describe, it, expect */
import getChangesetXML from '../../app/utils/get-changeset-xml'

describe('test changeset xml generation', () => {
  it('should return expected changeset xml for key-value pairs', () => {
    const xml = getChangesetXML({
      source: 'Observe',
      comment: 'test comment'
    })
    expect(xml).toMatchSnapshot()
  })
})
