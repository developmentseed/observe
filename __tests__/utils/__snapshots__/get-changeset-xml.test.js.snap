// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test changeset xml generation should return expected changeset xml for key-value pairs 1`] = `"<osm><changeset><tag k=\\"source\\" v=\\"Observe\\"/><tag k=\\"comment\\" v=\\"test comment\\"/></changeset></osm>"`;
